sorc/link_workflow.sh

```diff
@@ -384,34 +384,19 @@ if [[ -d "${HOMEgfs}/sorc/gsi_monitor.fd/install" ]]; then
   done
 fi
 
-# GDASApp
-if [[ -d "${HOMEgfs}/sorc/gdas.cd/build" ]]; then
-  declare -a JEDI_EXE=("gdas.x"
-    "gdas_soca_gridgen.x"
-    "gdas_soca_error_covariance_toolbox.x"
-    "gdas_fv3jedi_error_covariance_toolbox.x"
-    "gdas_soca_setcorscales.x"
-    "gdas_soca_diagb.x"
-    "fv3jedi_plot_field.x"
-    "gdasapp_chem_diagb.x"
-    "fv3jedi_fv3inc.x"
-    "fv3jedi_correction_increment.x"
-    "fv3jedi_ensemble_add_increment.x"
-    "gdas_ens_handler.x"
-    "gdas_incr_handler.x"
-    "gdas_obsprovider2ioda.x"
-    "gdas_socahybridweights.x"
-    "gdassoca_obsstats.x"
-    "gdasapp_land_ensrecenter.x"
-    "bufr2ioda.x"
-    "calcfIMS.exe"
-    "apply_incr.exe")
-  for gdasexe in "${JEDI_EXE[@]}"; do
-    if [[ -s "${gdasexe}" ]]; then
-        rm -f "${gdasexe}"
-    fi
-    ${LINK_OR_COPY} "${HOMEgfs}/sorc/gdas.cd/build/bin/${gdasexe}" .
-  done
+# GDASApp executables
+if [[ -d "${HOMEgfs}/sorc/gdas.cd/install" ]]; then
+  mv -v "${HOMEgfs}/sorc/gdas.cd/install/bin"/gdas*           ./
+  mv -v "${HOMEgfs}/sorc/gdas.cd/install/bin/bufr2ioda.x"     ./gdas_bufr2ioda.x
+  mv -v "${HOMEgfs}/sorc/gdas.cd/install/bin/calcfIMS.exe"    ./gdas_calcfIMS.x
+  mv -v "${HOMEgfs}/sorc/gdas.cd/install/bin/apply_incr.exe"  ./gdas_apply_incr.x
+fi
+
+# GDASApp libraries
+if [[ -d "${HOMEgfs}/sorc/gdas.cd/install" ]]; then
+  if [[ ! -d "${HOMEgfs}/lib" ]]; then mkdir "${HOMEgfs}/lib" || exit 1; fi
+  cd "${HOMEgfs}/lib" || exit 1
+  rsync -av "${HOMEgfs}/sorc/gdas.cd/install/lib/" ./
```

Comment:
```suggestion
  rsync -av -f "${HOMEgfs}/sorc/gdas.cd/install/lib/" ./
```
force over-writing of any previously copied `lib` contents.